framework_lib/src/chromium_ec/commands.rs

```diff
@@ -518,6 +518,37 @@ impl EcRequest<EcResponseUsbPdPowerInfo> for EcRequestUsbPdPowerInfo {
     }
 }
 
+// TODO: Actually 128, but if we go above ~80 EC returns REQUEST_TRUNCATED
```

Comment:
Huh.

I guess you can still use 128, but you need to use 2 host commands and update it in pieces.

Reply:
Done, can now send any number of LEDs. Tested by reducing this to 2 and it can still send 8 LEDs correctly.